tests/baselines/reference/exportNamespace11.errors.txt

```diff
@@ -0,0 +1,14 @@
+main.ts(2,52): error TS2339: Property 'Ghost' does not exist on type 'typeof import("intermediate")'.
+
+
+==== main.ts (1 errors) ====
+    import * as intermediate from './intermediate'
+    const ghost: intermediate.Ghost = new intermediate.Ghost()
+                                                       ~~~~~
+!!! error TS2339: Property 'Ghost' does not exist on type 'typeof import("intermediate")'.

```

Comment:
It would be nice to be able to give a better type-only-specific error message here, but this is consistent with existing error messages you can get with other ways of doing property access on a module with type-only exports.